common/decisions/wc_major_zul_decisions.txt

```diff
@@ -0,0 +1,102 @@
+﻿restore_empire_of_zul_decision = {
+	ai_check_interval = 120
+	major = yes
+	# PIC CREDIT https://phillipzhang.artstation.com/
+	picture = "gfx/interface/illustrations/decisions/zulian_restoration.dds"
+	desc = restore_empire_of_zul_decision_desc
+	selection_tooltip = restore_empire_of_zul_decision_tt
+
+	is_shown = {
+		culture = { has_cultural_pillar = heritage_zulite }
+		NOT = {
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:restore_empire_of_zul_decision
+			}
+		}
+		OR = {
+			title:e_amani.holder ?= root
+			title:e_drakkari.holder ?= root
+			title:e_gurubashi.holder ?= root
+			title:e_zandalari.holder ?= root
+			title:e_zulfarrak.holder ?= root
+		}
+	}
+
+	cost = {					
+		gold = 750
+		prestige = 1000
+	}
+	
+	is_valid = {
+		realm_size >= 40
+		prestige_level >= 4
+		calc_true_if = {
+			amount >= 4
+			title:e_amani.holder ?= root
+			title:e_drakkari.holder ?= root
+			title:e_gurubashi.holder ?= root
+			title:e_zandalari.holder ?= root
+			title:e_zulfarrak.holder ?= root
+		}
+		completely_controls = title:d_gundrak 			
+		completely_controls = title:d_drakmabwa 
+		completely_controls = title:d_hebdrakkar 
+		completely_controls = title:d_draktharon 
+		completely_controls = title:d_draksotra 
+		completely_controls = title:d_jagtar 
+		completely_controls = title:d_zuldazar 
+		completely_controls = title:d_mugamba 
+		completely_controls = title:d_mugamba 
+		completely_controls = title:d_nazmir 
+		completely_controls = title:d_torwatha 
+		completely_controls = title:d_zulaman 
+		completely_controls = title:d_jinthaalor 
+		completely_controls = title:d_zulgurub 
+		completely_controls = title:d_zulkunda 
+	}
+
+	is_valid_showing_failures_only = {
+		is_imprisoned = no
+		is_capable_adult = yes
+		is_at_war = no
+		is_independent_ruler = yes
+		religion.faith = { has_doctrine = special_doctrine_is_troll_faith }
+	}
+
+	effect = {
+		# Can only happen once
+		hidden_effect = {
+			add_to_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:restore_empire_of_zul_decision
+			}
+		}
+		custom_tooltip = restore_empire_of_zul_decision_tt
+		save_scope_as = founder
+
+		create_empire_of_zul_effect = yes
+
+		# Notification
+		every_player = {
+			limit = {
+				OR = {
+					culture = { has_cultural_pillar = heritage_zulite }
+					is_within_diplo_range = { CHARACTER = scope:founder }
+				}
+			}
+			trigger_event = zul_restoration.0001
+		}
+		
+		add_prestige_level = 2
+		add_trait = augustus
```

Comment:
For gorian empire we did some unique name for this trait. Zulustus? idk, you can edit it in `00_trait.txt` file